scraper.rb

```diff
@@ -3,11 +3,29 @@
 # frozen_string_literal: true
 
 require 'scraperwiki'
+require 'scraped'
+require 'pry'
+require_rel 'lib'
 
-require_relative 'lib/members.rb'
+require 'open-uri/cached'
+OpenURI::Cache.cache_path = '.cache'
 
-members_url = 'http://api.openhluttaw.org/en/memberships'
+def scrape(h)
+  url, klass = h.to_a.first
+  klass.new(response: Scraped::Request.new(url: url).response)
+end
 
 ScraperWiki.sqliteexecute('DELETE FROM data') rescue nil
-data = Members.new(members_url).members_of_the_lower_house.map(&:to_h)
+
+def scrape_lower_house_member_records(url, data = [])
+  member_records = (scrape url => MemberRecords)
+  data += member_records.members_of_the_lower_house.map(&:to_h)
+
+  return data if (next_page = member_records.next).nil?
+  scrape_lower_house_member_records(next_page, data)
+end
+
+start_url = 'http://api.openhluttaw.org/en/memberships'
+data = scrape_lower_house_member_records(start_url)
+# data.each { |d| puts d.reject { |_k, v| v.to_s.empty? }.sort_by { |k, _v| k }.to_h }
 ScraperWiki.save_sqlite([:id], data)
```

Comment:
I could add an additional key but that would make the before/after output differ. It looks like there should be a unique id per membership. http://api.openhluttaw.org/en/memberships?page=2

There don't appear to be any duplicates:
```Ruby
> data.length == data.map { |d| d[:id] }.uniq.length
=> true
```